src/test/java/uk/gov/dhsc/htbhf/smartstub/helper/PersonTestFactory.java

```diff
@@ -0,0 +1,31 @@
+package uk.gov.dhsc.htbhf.smartstub.helper;
+
+import uk.gov.dhsc.htbhf.smartstub.model.PersonDTO;
+
+import static uk.gov.dhsc.htbhf.smartstub.helper.AddressDTOTestDataFactory.aValidAddress;
+import static uk.gov.dhsc.htbhf.smartstub.helper.TestConstants.HOMER_DATE_OF_BIRTH;
+import static uk.gov.dhsc.htbhf.smartstub.helper.TestConstants.HOMER_FIRST_NAME;
+import static uk.gov.dhsc.htbhf.smartstub.helper.TestConstants.SIMPSON_LAST_NAME;
+
+/**
+ * Test data factory for {@link PersonDTO} objects.
+ */
+public class PersonTestFactory {
+
+    private static final String NINO = "EB123456C";
+
+    public static PersonDTO aValidPerson() {
+        final String nino = "EA000000C";
```

Comment:
Why does this need a different NINO to `buildDefaultPerson` ?

Reply:
It doesn't. It was copied and pasted from smart-stub, not sure why it was in there. I've removed it.